lib/syskit/telemetry/ui/name_service.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Syskit
+    module Telemetry
+        module UI
+            # Copy of Runkit's local name service to use with orocos.rb
+            class NameService < Orocos::NameServiceBase
+                # A new NameService instance
+                #
+                # @param [Hash<String,Orocos::TaskContext>] tasks The tasks which are
+                #        known by the name service.
+                # @note The namespace is always "Local"
+                def initialize(tasks = [])
+                    @registered_tasks = Concurrent::Hash.new
+                    tasks.each { |task| register(task) }
+                end
+
+                def names
+                    @registered_tasks.keys
+                end
+
+                def include?(name)
+                    puts @registered_tasks.keys
```

Comment:
This `puts` doesnt seem intentional

Reply:
It was already removed on my branch ... not sure what happened here.